lib/matplotlib/mpl-data/stylelib/_mpl-gallery.mplstyle

```diff
@@ -16,4 +16,4 @@ ytick.major.size: 0.0
 
 # colors:
 image.cmap    : Blues
-# axes.prop_cycle: cycler('color', ['FF7F0E', '1F77B4', '2CA02C'])
+axes.prop_cycle: cycler('color', ['1f77b4', '82bbdb', 'ccdff1'])
```

Comment:
This first color should be exactly the same as our current C0: `+#1f7764`

Reply:
Um it is '1f77b4'
https://github.com/matplotlib/matplotlib/blob/b05b8f00978096b86d81a3106c64a23e8c87eef4/lib/matplotlib/_color_data.py#L15